### ISSUE TITLE
Word Embeddings not Accuracte

### ISSUE THREAD

**Issue by Aiman-Shivani** [tensorflow/tensorflow#50798]
I am trying to build my own word2vec model using the code provided here
Link: - https://www.tensorflow.org/tutorials/text/word2vec

So i have even tried to increase the data as well for training the word embedding and i am able to achieve a good model accuracy but when i plot the word vectors on the Embedding Projector the distance between words or the word similarity is really bad, if i even use the cosine distance formula between very similar words the result is bad.

Whereas if the same data is used to train own embeddings using the Gensim library ( not pre-trained) the results of distance and similarity are way better, even on the Embedding Projector as well.

Please can someone help me regarding this, i want to use the Word2Vec code only which is provided by TensorFlow but i am not able to get good results for word distance and word similarity.

**Comment by ymodak** [comment]
duplicate #https://github.com/tensorflow/tensorflow/issues/50645
